### PR TITLE
Fix #2841: Update the documentation on oppia.github.io

### PR DESCRIPTION
**Pull request by terrameijar**
Updated the pages/Screenshots.html page

### CODE REVIEW

File: pages/Screenshots.html [hunk 1]
```diff
@@ -49,7 +49,7 @@
     <a id="my-explorations-dashboard" class="anchor" href="#my-explorations-dashboard" aria-hidden="true">
       <span class="octicon octicon-link"></span>
     </a>
-    "My Explorations" dashboard
+    Creator Dashboard
```
Comment (seanlip) [inline pages/Screenshots.html]: Do you still need this section? I think we can remove it altogether...

**Review by seanlip — CHANGES_REQUESTED** [review]
Thanks, just one comment, otherwise looks good!

File: pages/Screenshots.html [hunk 2]
```diff
@@ -25,9 +25,9 @@
     <a id="gallery" class="anchor" href="#gallery" aria-hidden="true">
       <span class="octicon octicon-link"></span>
     </a>
-    Gallery
+    Library
   </h3>
-  <p><img src="static/images/gallery.png" alt="Screenshot of gallery page" class="screenshot-image"></p>
+  <p><img src="static/images/library.png" alt="Screenshot of gallery page" class="screenshot-image"></p>
```
Comment (seanlip) [inline pages/Screenshots.html]: Alt text should be changed, too.

Are you planning to delete the old image? (Only do so if we don't use it elsewhere.)

**Review by seanlip — COMMENTED** [review]
Thanks! Sorry, saw just one more thing.

File: pages/Screenshots.html (hunk 2 above)
Comment (terrameijar) [reply pages/Screenshots.html]: Yes I will delete the old image but not right now, it is in use in at least one other location so I will only delete it once I have updated the other page that contains this image.

**Review by seanlip — APPROVED** [review]

**Comment by seanlip** [comment]
Sounds good, thanks! Merging.

(Also, please feel free to submit follow-up PRs to update other parts of the site. Thanks!)

**Comment by terrameijar** [comment]
This was my first PR and I'm excited about getting it merged. I will get back to updating other parts of the site after I'm done doing cartwheels. Thank you!

**Comment by seanlip** [comment]
Ah yes, I meant to congratulate you on your first PR, but forgot! Better late than never, though, so congratulations, and thanks :-)

**Pull request merged**